cogs/track_react.py

```diff
@@ -88,6 +90,16 @@ def parse_codeblocks(message: str) -> str:
                 lines.append(line)
         return "\n".join(lines)
 
+    @staticmethod
+    def parse_url(message: str) -> str:
+        """Return a message with URLs removed."""
+        lines = []
```

Comment:
Do you need to process this on a per line basis? Can you do the re.sub on the message as a single string, using a single call?

Splitting, searching, building a list, joining, etc is extra steps that shouldn't be needed. A single sub function call should suffice.